client/ayon_core/pipeline/context_tools.py

```diff
@@ -587,6 +587,7 @@ def version_up_workfile():
     host = registered_host()
     current_file = host.get_current_workfile()
     if not current_file:
+        log.debug("Current file not found.")
```

Comment:
Current file not found seems quite warning for the user - did someone delete it? Also, Maya may have a filename set but the file not existing anymore - we're not checking that in `host.get_current_workfile`

I suppose the right wording is along the lines of:
```suggestion
        log.debug(
            "Current file is unsaved and can't be versioned up. "
            "Please save your file first.")
```

Also, we'll need to make sure the debug messages show up to the artists in one way or another that an artist may actually notice (I'm not sure if they do by default in all hosts, and in this case in Maya)?